backend/scraper/mapping_police_violence.py

```diff
@@ -162,12 +167,26 @@ def make_all_tables():
         sub_table = full_df
         table_list.append(sub_table)
     table_dict = {table_names[i]: table_list[i] for i in range(len(table_list))}
+    df_victim = table_dict["victim"]
```

Comment:
So this is filtering the rows by removing rows with duplicate victim names, while retaining all rows with unknown victim names? 

- This would be a great thing to move into a separate deduplicate_rows function to hide these implementation details and express the intent more clearly. It would take the table_dict and update each dataframe with the filtered index. 
- I would just inline the victim_indices_* and rename the filtered_indices on line 174 to `valid_full_names`. 
- Consider renaming `df_victim_filtered` to something more descriptive like `deduplicated_by_name`
- You can use `~` to negate `valid_full_names` and save some typing: `deduplicated_by_name = np.concat([df_victim[~valid_full_names], df_victim[valid_full_names].drop_duplicates(...)])`